common/src/chain/pos.rs

```diff
@@ -24,3 +29,56 @@ pub type PoolId = Id<Pool>;
 #[derive(Eq, PartialEq, TypeName)]
 pub enum Delegation {}
 pub type DelegationId = Id<Delegation>;
+
+#[derive(Debug, Clone, PartialEq, Eq, Ord, PartialOrd)]
+pub struct PoSChainConfig {
+    /// The lowest possible difficulty
+    target_limit: Uint256,
+    /// Time interval between the blocks targeted by the difficulty adjustment algorithm
+    target_block_time: Duration,
```

Comment:
Now that I think about it, there's a concern that data will be lost since we use this strictly as seconds. Someone can do `from_millis(500)`, which will return zero for seconds.

We need this to be strictly seconds.